dbt_project.yml

```diff
@@ -1423,4 +1423,9 @@ models:
     openxswap:
       +schema: openxswap
       optimism:
-        +schema: openxswap_optimism
\ No newline at end of file
+        +schema: openxswap_optimism
+
+    moola:
+      +schema: moola
+      celo:
+        +schema: moola_celo
```

Comment:
Since `moola_celo` is added in the config blocks, we can recover this file.

We are looking to eventually clean up the project file a bit by removing schema definitions there, rather put the schema name in the model config block itself.